.github/workflows/sbom.yml

```diff
@@ -0,0 +1,71 @@
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      dotcms_version:
```

Comment:
Should not be for one version only - any version release would need to have SBOM